Xamarin.Forms.Platform.UAP/SwipeViewRenderer.cs

```diff
@@ -0,0 +1,484 @@
+﻿using System;
+using System.Linq;
+using System.ComponentModel;
+using System.Collections.Generic;
+using WSwipeBehaviorOnInvoked = Microsoft.UI.Xaml.Controls.SwipeBehaviorOnInvoked;
+using WSwipeControl = Microsoft.UI.Xaml.Controls.SwipeControl;
+using WSwipeItems = Microsoft.UI.Xaml.Controls.SwipeItems;
+using WSwipeItem = Microsoft.UI.Xaml.Controls.SwipeItem;
+using WSwipeMode = Microsoft.UI.Xaml.Controls.SwipeMode;
+using System.Collections.Specialized;
+
+namespace Xamarin.Forms.Platform.UWP
+{
+	public class SwipeViewRenderer : ViewRenderer<SwipeView, WSwipeControl>
+	{
+		bool _isDisposed;
+		Dictionary<WSwipeItem, SwipeItem> _leftItems;
+		Dictionary<WSwipeItem, SwipeItem> _rightItems;
+		Dictionary<WSwipeItem, SwipeItem> _topItems;
+		Dictionary<WSwipeItem, SwipeItem> _bottomItems;
+
+		public SwipeViewRenderer()
+		{
+			AutoPackage = false;
+		}
+
+		protected override void OnElementChanged(ElementChangedEventArgs<SwipeView> e)
+		{
+			base.OnElementChanged(e);
+
+			if (e.NewElement != null)
+			{
+				e.NewElement.CloseRequested += OnCloseRequested;
+
+				if (Control == null)
+				{
+					SetNativeControl(new WSwipeControl());
+
+					Element.LeftItems.CollectionChanged += OnSwipeItemsChanged;
```

Comment:
Subscribe to these outside the Control check. The Element on this might get swapped out but the Control will stay the same. Unsubscribe from the OldElement if OldElement is not null